src/chrome/content/rules/UPS.com.xml

```diff
@@ -22,28 +22,42 @@
 <ruleset name="UPS.com (partial)">
 
 	<target host="ups.com" />
-	<target host="*.ups.com" />
+		<exclusion pattern="^http://ups.com/$" />
+	<target host="www.ups.com" />
+		<exclusion pattern="^http://www.ups.com/$" />
+	<target host="wwwapps.ups.com" />
+	<target host="www.apps.ups.com" />
+	<target host="campusship.ups.com" />
+	<target host="www.campusship.ups.com" />
+
 		<!--
 			Redirects to http:
 						-->
-		<!--exclusion pattern="^http://www\.ups\.com/content/globalentry$" /-->
-		<!--
-			all ur pkgs r now belong to nsa
+		<exclusion pattern="^http://www\.ups\.com/content/globalentry$" />
+			<test url="http://www.ups.com/content/globalentry" />
 
-			(Now redirects to login:
-						-->
-		<!--exclusion pattern="^http://www\.ups\.com/WebTracking/track($|\?)" /-->
 		<!--
-			Exceptions:
-					-->
-		<exclusion pattern="^http://www\.ups\.com/+(?!assets/|favicon\.ico|img/|styles/|stylesheets/)" />
-
+			Now redirects to login:
+						-->
+		<exclusion pattern="^http://www\.ups\.com/WebTracking/track($|\?)" />
+			<test url="http://www.ups.com/WebTracking/track" />
+			<test url="http://www.ups.com/WebTracking/track?" />
```

Comment:
The exclusions were already there just commented out, unfortunately I don't have UPS tracking number handy.

I did try `123456`

Linkname: UPS: Tracking Information
    URL: http://wwwapps.ups.com/WebTracking/track
Post Data:
loc=en_US&HTMLVersion=5.0&USER_HISTORY_LIST=&trackNums=123456&track.x=Track
Post Content Type: application/x-www-form-urlencoded